### DISCUSSION TITLE
ChatGPT for tvision

### DISCUSSION THREAD

**Discussion by matushromada** [magiblot/tvision#164] [Show and tell]
I have created a ChatGPT Assistant for this great port of tvision
https://chatgpt.com/g/g-T1OsW2aMu-tvision-coding-assistant

It's loaded with original Turbo Vision 2.0 knowledge and told to pay attention to changes in this particular port.

**Comment by magiblot** [comment]
Hi Matúš! Thank you, this sounds very interesting. However, when I follow the link, I get a 404 page:

![Screenshot_20240915_212207](https://github.com/user-attachments/assets/cfc5fba5-4c1c-43b4-9574-4b83da0787f6)

**Comment by matushromada** [comment]
Thanks for noticing me, there was a bug in GPT when saving permissions. Now the link works. I was able to create an application with it, with zero C++ and Turbo Vision knowledge.

**Comment by matushromada** [comment]
The assistant was loaded with best practices when using the Turbo Vision library and asked to adhere to them when creating the code. Give it a try :)
